pkgs/development/python-modules/django-bootstrap5/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, beautifulsoup4
+, buildPythonPackage
+, django
+, fetchFromGitHub
+, pillow
+, pythonOlder
+, python
+, setuptools
+, hatchling
+}:
+
+buildPythonPackage rec {
+  pname = "django-bootstrap5";
+  version = "23.3";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "zostera";
+    repo = "django-bootstrap5";
+    rev = "v${version}";
+    hash = "sha256-FIwDyZ5I/FSaEiQKRfanzAGij86u8y85Wal0B4TrI7c=";
+  };
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace "\"Framework :: Django :: 4.2\"," ""
```

Comment:
Do we require a new version of poetry-core or hatchling for this to work? Do we have this on staging-next already?

Reply:
dunno, it works for this version, might be an old hatchling version tho